data/udisks2.service.in

```diff
@@ -3,10 +3,27 @@ Description=Disk Manager
 Documentation=man:udisks(8)
 
 [Service]
-Type=dbus
 BusName=org.freedesktop.UDisks2
+CapabilityBoundingSet=CAP_SYS_RAWIO
 ExecStart=@udisksdprivdir@/udisksd
+IPAddressDeny=any
 KillSignal=SIGINT
+LimitMEMLOCK=0
+LockPersonality=yes
+MemoryDenyWriteExecute=yes
+NoNewPrivileges=yes
+PrivateNetwork=yes
+PrivateTmp=yes
+ProtectHome=yes
+ProtectSystem=strict
```

Comment:
Plugins may need write access to `/run`, `/var` and possibly the homedir. Please check what exactly this option limits.

Reply:
[systemd.exec(5)](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#ProtectSystem=):
```
Takes a boolean argument or the special values "full" or "strict". If true, mounts the /usr and /boot
           directories read-only for processes invoked by this unit. If set to "full", the /etc directory is
           mounted read-only, too. If set to "strict" the entire file system hierarchy is mounted read-only,
           except for the API file system subtrees /dev, /proc and /sys (protect these directories using
           PrivateDevices=, ProtectKernelTunables=, ProtectControlGroups=).
```
To allow /run and /var, I'd propose `ProtectSystem=full`, so only /boot, /usr and /etc are read-only.